PCLConfig.cmake.in

```diff
@@ -112,14 +112,11 @@ macro(find_boost)
   endif()
 endmacro()
 
-#remove this as soon as eigen is shipped with FindEigen.cmake
-macro(find_eigen)
+macro(find_eigen3)
   if(PCL_ALL_IN_ONE_INSTALLER)
-    set(EIGEN_ROOT "${PCL_ROOT}/3rdParty/Eigen")
-  elseif(NOT EIGEN_ROOT)
-    get_filename_component(EIGEN_ROOT "@EIGEN_INCLUDE_DIRS@" ABSOLUTE)
+    set(Eigen3_DIR "${PCL_ROOT}/3rdParty/Eigen3/share/eigen3/cmake/")
   endif()
-  find_package(Eigen 3.3)
+  find_package(Eigen3 3.3 REQUIRED NO_MODULE)
```

Comment:
This should be `find_dependency` instead of `find_package`. (In other places, too)

Reply:
You're probably right, but I guess this should be consistently handled for all package dependencies in a separate PR. 

I'm anyways not sure about the state of this PR. I solved the last finding 4 months ago, but so far I got no approvals. Not sure if this will ever get merged..